Goal: Fix some wording in su(1)

Fixes: 

Note: 

Status wrt upstream: Applied in CVS

Index: shadow-4.0.18.1/man/su.1.xml
===================================================================
--- shadow-4.0.18.1.orig/man/su.1.xml	2007-06-17 07:30:14.492480422 +0200
+++ shadow-4.0.18.1/man/su.1.xml	2007-06-17 07:31:15.989379347 +0200
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <refentry id='su.1'>
-  <!--  $Id: su.1.xml,v 1.26 2006/07/18 17:02:01 kloczek Exp $ -->
+  <!--  $Id: su.1.xml,v 1.30 2006/11/12 19:20:34 kloczek Exp $ -->
   <refmeta>
     <refentrytitle>su</refentrytitle>
     <manvolnum>1</manvolnum>
@@ -8,7 +8,7 @@
   </refmeta>
   <refnamediv id='name'>
     <refname>su</refname>
-    <refpurpose>change user ID or become super-user</refpurpose>
+    <refpurpose>change user ID or become superuser</refpurpose>
   </refnamediv>
   <refsynopsisdiv id='synopsis'>
     <cmdsynopsis>
@@ -27,10 +27,10 @@
   <refsect1 id='description'>
     <title>DESCRIPTION</title>
     <para>
-      <command>su</command> is used to become another user during a login
-      session. Invoked without a <option>username</option>, 
+      The <command>su</command> command is used to become another user during
+      a login session. Invoked without a <option>username</option>, 
       <command>su</command> defaults to
-      becoming the super user. The optional argument <option>-</option> may
+      becoming the superuser. The optional argument <option>-</option> may
       be used to provide an environment similar to what the user would
       expect had the user logged in directly.
     </para>
@@ -58,7 +58,7 @@
       The current environment is passed to the new shell. The value of
       <envar>$PATH</envar> is reset to <filename>/bin:/usr/bin</filename>
       for normal users, or <filename>/sbin:/bin:/usr/sbin:/usr/bin</filename>
-      for the super user. This may be changed with the
+      for the superuser. This may be changed with the
       <emphasis>ENV_PATH</emphasis> and <emphasis>ENV_SUPATH</emphasis>
       definitions in <filename>/etc/login.defs</filename>.
     </para>
@@ -78,7 +78,7 @@
       <varlistentry>
 	<term>
 	  <option>-c</option>, <option>--command</option>
-	  <replaceable>SHELL</replaceable>
+	  <replaceable>COMMAND</replaceable>
 	</term>
 	<listitem>
 	  <para>
@@ -112,10 +112,10 @@
 	<listitem>
 	  <para>The shell that will be invoked.</para>
 	  <para>
-	    The invoked shell is choosen among (higest priority first):
+	    The invoked shell is chosen from (highest priority first):
 	    <itemizedlist>
 	      <listitem>
-		<para>The shell specified with --shell</para>
+		<para>The shell specified with --shell.</para>
 	      </listitem>
 	      <listitem>
 		<para>
@@ -141,10 +141,10 @@
 	  <para>
 	    If the target user has a restricted shell (i.e. the shell field of
 	    this user's entry in <filename>/etc/passwd</filename> is not
-	    specified in <filename>/etc/shell</filename>), then the
+	    listed in <filename>/etc/shell</filename>), then the
 	    <option>--shell</option> option or the <envar>$SHELL</envar>
-	    environment variable won't be taken into account unless
-	    <command>su</command> is called by the root.
+	    environment variable won't be taken into account, unless
+	    <command>su</command> is called by root.
 	  </para>
 	</listitem>
       </varlistentry>
